code/modules/virus2/effect.dm

```diff
@@ -1090,6 +1090,34 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
 		to_chat(mob, "Suddenly, your knowledge of languages comes back to you.")
 
 
+/datum/disease2/effect/gregarious
+	name = "Gregarious Impetus"
+	stage = 4
+	max_chance = 25
+	max_multiplier = 4
+
+/datum/disease2/effect/gregarious/activate(var/mob/living/carbon/mob)
+	var/others_count = 0
+	for(var/mob/living/carbon/m in oview(5, mob))
+		if (airborne_can_reach(mob.loc, m.loc, 9)) // Apparently mobs physically block airborne viruses
+			others_count += 1
+	if (others_count >= multiplier)
+		to_chat(mob, "<span class='notice'>A friendly sensation is satisfied with how many are near you - for now.</span>")
+		mob.adjustCloneLoss(-multiplier * 2)
+		mob.reagents.add_reagent(PARACETAMOL, multiplier) // The hug chemical for memes
```

Comment:
I still think you should replace this with something better. Is there nothing else you think would be a better reward?